### PR TITLE
Remove JQuery based element from contextNavigation

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: app/assets/javascripts/arclight/context_navigation.js [hunk 1]
```diff
@@ -101,9 +101,9 @@ class Placeholder {
 
 class ContextNavigation {
   constructor(el, originalParents = null, originalDocument = null) {
-    this.el = $(el);
-    this.data = this.el.data();
-    this.parentLi = this.el.parent();
+    this.el = el
+    this.data = $(this.el).data();
```
Comment (thatbudakguy) [inline app/assets/javascripts/arclight/context_navigation.js:105]: can we not do `this.el.dataset` here?

Comment (jcoyne) [reply app/assets/javascripts/arclight/context_navigation.js:105]: That's already done in #1210

Comment (thatbudakguy) [reply app/assets/javascripts/arclight/context_navigation.js:105]: got it. I think if this gets rebased and conflicts are resolved it'll be a little easier to review

Comment (jcoyne) [reply app/assets/javascripts/arclight/context_navigation.js]: It has been rebased.

**Review by thatbudakguy — APPROVED** [review]

**Pull request merged**